src/EscapeAnalysis.jl

```diff
@@ -323,6 +317,15 @@ function find_escapes(ir::IRCode, nargs::Int)
     return state
 end
 
+function update_non_bitstype_changes!(args::Vector{Any}, ir::IRCode, @nospecialize(info::EscapeInformation), changes::Changes)
```

Comment:
```suggestion
function add_changes!(args::Vector{Any}, ir::IRCode, @nospecialize(info::EscapeInformation), changes::Changes)
```
I think this name is better to be more general ?